scripts/integration_test_subsets.py

```diff
@@ -0,0 +1,48 @@
+"""Defines test subsets of integration tests.
```

Comment:
Ah interesting, so it's regex-based. 

Earlier I was looking at actually moving Go files around so these tests sit in separately named packages. I don't recall why I didn't finish that up but it certainly looked doable.

Reply:
I refrained because there are an awful lot of relative paths, didn't feel confident (at that point) moving these to separate packages.

Reply:
Yeah, understood.